src/Compilers/CSharp/Test/Syntax/Syntax/SyntaxNodeCacheTests.cs

```diff
@@ -0,0 +1,78 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#nullable disable
```

Comment:
Nit: why disable this new test file?

Reply:
For tests, in general, it seems useful to allow nullable mismatches. That said, I don't know if there any such cases in this test file currently.

Reply:
I also do not find nullable analysis in unit-test files useful. To be precise, I even find it distracting, i.e. getting in a way.